.gitignore

```diff
@@ -1,6 +1,7 @@
 # compiled output
 /dist
 /node_modules
+polymesh-rest-api-swagger-spec.json
```

Comment:
This maybe could go in `/dist`, but not sure how that affects the pipe line.

Reply:
This should NOT go into `/dist` its ~300KB and would be bloat in an image / package.